web/cobrands/sass/_base.scss

```diff
@@ -712,15 +712,23 @@ ul.error {
 }
 
 // this is a skip to nav for mobile users only
+label[for="main-nav-btn"] {
+  font-weight: normal;
+  margin: 0;
+}
+
 #nav-link {
   width: 3em;
   height: 0;
   padding-top: 3em;
   overflow: hidden;
-  background: transparent url(../fixmystreet/images/#{$menu-image}.png) center center no-repeat;
-  background-image: inline-image("../fixmystreet/images/#{$menu-image}.svg"), none;
+  background: transparent inline-image("../fixmystreet/images/#{$menu-image}.svg") center center no-repeat;
   background-size: 22px 18px;
   #{$right}: 0;
+
+  html.js-nav-open & {
+    background-image: inline-image("../fixmystreet/images/cross-grey.svg");
```

Comment:
Should really make a custom SVG for this, that matches the `menu-*.svg` images in `/web/cobrands/fixmystreet/images/`. I’ll sort that.

Reply:
Also, aside from this line, it looks like `cross-grey.svg` isn’t currently used anywhere, so maybe we should just delete it.

Reply:
Following up on my own comment – `cross-grey.svg` and `cross-grey.png` were added in bdaf17a566b235865ad004e427c6f2230df39f24 for toggling the visibility of the `#report-a-problem-sidebar`, and then all references to the two files were removed in `44c97f31635cda9cdd0ce2d37d186bd5af487e0c`, when we consolidated all of the inspector/admin-facing button icons into a single sprite. But for some reason I didn’t remove the `cross-*` image files. So I’m going to add a commit that removes them now. :+1: